lego/apps/users/fixtures/initial_abakus_groups.yaml

```diff
@@ -3,143 +3,462 @@
 #
 - model: users.abakusgroup
   pk: 1
-  fields: {deleted: false, name: Users, description: "Brukere p\xE5 Abakus.no", contact_email: '',
```

Comment:
Move this to separate pr pls

Reply:
This is just a change in how the yaml is generated. So really it only adds one group, which is needed for the tests.

Reply:
But why is the formatting changed?

Reply:
It's impossible to read the diff as is

Reply:
I'm guessing some dependency got updated. I moved the regeneration of the formatting to a separate commit now.